### PR TITLE
CNV-29192-29107: RN Real-time checkup and cluster config

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.16

Issue: [CNV-29192](https://issues.redhat.com//browse/CNV-29192)
[CNV-29170](https://issues.redhat.com//browse/CNV-29170)

Link to docs preview: https://77076--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-16-release-notes.html#virt-4-16-virtualization

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: virt/release_notes/virt-4-16-release-notes.adoc [hunk 1]
```diff
@@ -68,9 +68,16 @@ This release adds new features and enhancements related to the following compone
...
+//CNV-29170 Real-time cluster configuration
+* You can xref:../../virt/virtual_machines/advanced_vm_management/virt-configuring-cluster-realtime-workloads.adoc#virt-configuring-cluster-realtime-workloads[configure an {product-title} cluster] to run real-time virtual machine (VM) workloads that require low and predictable latency.
+
+
 [id="virt-4-16-networking"]
 === Networking
 
+//CNV-29192 Running real-time checkup
+* You can now run a predefined xref:../../virt/monitoring/virt-running-cluster-checkups.adoc#virt-running-real-time-checkup_virt-running-cluster-checkups[checkup] to verify that your {product-title} cluster is configured to run virtualized real-time workloads.
+
```
Comment (orelmisan) [inline virt/release_notes/virt-4-16-release-notes.adoc:80]: I think this should also be under "Virtualization".

**Review by orelmisan — APPROVED** [review]
Thank you

**Review by cbippley — COMMENTED** [review]
LGTM!

**Pull request merged**